plugins/catalog/report.api.md

```diff
@@ -40,11 +40,10 @@ import { TableProps } from '@backstage/core-components';
 import { TabProps } from '@material-ui/core/Tab';
 import { UserListFilterKind } from '@backstage/plugin-catalog-react';
 
+// Warning: (ae-forgotten-export) The symbol "InternalAboutCardProps" needs to be exported by the entry point index.d.ts
+//
 // @public
-export interface AboutCardProps {
-  // (undocumented)
-  variant?: InfoCardVariants;
-}
+export type AboutCardProps = Pick<InternalAboutCardProps, 'variant'>;
```

Comment:
Got a warning here, I think the best workaround is to keep declaring duplicate `AboutCardProps`